packages/core/src/index.scss

```diff
@@ -1,3 +1,3 @@
+@forward '~@onfido/castor-tokens'; // TODO: remove after Castor and Castor Tokens split BREAKING CHANGE
```

Comment:
Is the goal that people import tokens and Color types directly from `castor-tokens` instead?

I don't mind leaving this here permanently, as a re-export.

Reply:
I would expect this, yes. It would be a breaking change. But needs more thinking here..

Reply:
Alright we'll cross that bridge when we get there.
I personally prefer importing all of it from `@onfido/castor`.